tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -22,22 +22,18 @@
 from __future__ import division
 from __future__ import print_function
 
-import collections
 import csv
 
 import six
 from six import StringIO
 from werkzeug import wrappers
-import numpy as np
 
 from tensorboard import errors
 from tensorboard import plugin_util
 from tensorboard.backend import http_util
 from tensorboard.data import provider
 from tensorboard.plugins import base_plugin
 from tensorboard.plugins.scalar import metadata
-from tensorboard.util import tensor_util
```

Comment:
Also remove `//tensorboard/util:tensor_util` from `:scalars_plugin`?

Reply:
I think I got everything now. Verified for sure that tests definitely pass locally for me.

Reply:
Done